src/PrestaShopBundle/Controller/Admin/AdvancedParameters/AdministrationController.php

```diff
@@ -0,0 +1,111 @@
+ */
+
+namespace PrestaShopBundle\Controller\Admin\AdvancedParameters;
+
+use PrestaShopBundle\Controller\Admin\FrameworkBundleAdminController;
+use PrestaShopBundle\Security\Voter\PageVoter;
+use Symfony\Component\Form\FormInterface;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * Responsible of "Configure > Advanced Parameters > Administration" page display
+ */
+class AdministrationController extends FrameworkBundleAdminController
+{
+    const CONTROLLER_NAME = 'AdminAdminPreferences';
+
+    /**
+     * @var FormInterface
+     * @return Response
+     */
+    public function indexAction(FormInterface $form = null)
+    {
+        $form = is_null($form) ? $this->get('prestashop.adapter.administration.form_handler')->getForm() : $form;
+
+        $twigValues = array(
+            'layoutHeaderToolbarBtn' => array(),
+            'layoutTitle' => $this->get('translator')->trans('Administration', array(), 'Admin.Navigation.Menu'),
+            'requireAddonsSearch' => true,
+            'requireBulkActions' => false,
+            'showContentHeader' => true,
+            'enableSidebar' => true,
+            'help_link' => $this->generateSidebarLink('AdminAdminPreferences'),
+            'requireFilterStatus' => false,
+            'form' => $form->createView(),
+        );
+
+        return $this->render('PrestaShopBundle:Admin/AdvancedParameters:administration.html.twig', $twigValues);
+    }
+
+    /**
+     * @return RedirectResponse
+     */
+    public function processFormAction(Request $request)
+    {
+        if ($this->isDemoModeEnabled()) {
+            $this->addFlash('error', $this->getDemoModeErrorMessage());
+
+            return $this->redirectToRoute('admin_performance');
+        }
+
+        $this->dispatchHook('actionAdminAdminPreferencesControllerPostProcessBefore', array('controller' => $this));
+        $form = $this->get('prestashop.adapter.administration.form_handler')->getForm();
+        $form->handleRequest($request);
+
+        if (!in_array(
+            $this->authorizationLevel($this::CONTROLLER_NAME),
+            array(
+                PageVoter::LEVEL_READ,
+                PageVoter::LEVEL_UPDATE,
+                PageVoter::LEVEL_CREATE,
+                PageVoter::LEVEL_DELETE,
+            )
+        )) {
+            $this->addFlash('error', $this->trans('You do not have permission to update this.', 'Admin.Notifications.Error'));
+
+            return $this->redirectToRoute('admin_administration');
+        }
+
+        if ($form->isSubmitted()) {
+            $data = $form->getData();
+
+            $saveErrors = $this->get('prestashop.adapter.administration.form_handler')->save($data);
+
+            if (0 === count($saveErrors)) {
+                $this->addFlash('success', $this->trans('Successful update.', 'Admin.Notifications.Success'));
```

Comment:
Shouldn't it be "Update successful"? We have both options in the project, what is the best one? (ping @LouiseBonnard)